src/docs/README.md

```diff
@@ -1,90 +0,0 @@
-# Table of Contents
```

Comment:
this file was deleted because Docusaurus will render all Markdown files in `docs/` as pages, so this was being rendered as a page.